css/answers/gopal-q10.html

```diff
@@ -0,0 +1,60 @@
+<html>
+    <head>
+
+        <!-- Global site tag (gtag.js) - Google Analytics -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157837769-1"></script>
+        <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            CSS | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="Which command is used for the selection of all the elements of a paragraph?">
+    </head>
+    <body>
+
+        <div>
+            <img src="../../assets/CN_logo.png" height='72px' alt="Coding Ninjas Logo">
+        </div>
+
+        <div>
+            <p>
+                <small>
+
+                    <a href="../../index.html">Home</a>
+                    &gt; <a href="../index.html">CSS Questions</a>
+                    &gt; Which command is used for the selection of all the elements of a paragraph?
+
+                </small>
+               
+            </p>
+        </div>
+
+
+        <div>
+            <div>
+                <h4>
+                    <h4>Which command is used for the selection of all the elements of a paragraph?</h4>
+                </h4>
+            </div>
+            <br>
+            <br>
+            <div>
+                <strong>Answer:</strong> 
+                <p>
+                    The p[lang] command is used for selecting all the elements of a paragraph. 
```

Comment:
Please give an example or code snippet to demonstrate.

Reply:
<!DOCTYPE html>
<html>
<head>
<style>
p:lang(it) { 
  background: yellow;
}
</style>
</head>
<body>

<p>I live in Italy.</p>
<p lang="it">Ciao bella!</p>

<p><b>Note:</b> For :lang to work in IE8, a DOCTYPE must be declared.</p>

</body>
</html>

Reply:
Please add this in the file itself from your editor (like you did before).